modules/virt-example-nmstate-IP-management.adoc

```diff
@@ -51,6 +51,11 @@ The following snippet ensures that the interface has no IP address:
 # ...
 ----
 
+[IMPORTANT]
+====
+Always set the `state` parameter to `up` when you set the `ipv4.enabled` or the `ipv6.enabled` parameter to `false`. If you set `state: down` with this configuration, the interface receives a DHCP IP address because of automatic DHCP assignment. 
```

Comment:
For the first sentence, let's say:

Always set the `state` parameter to `up` when you set the `ipv4.enabled` and the `ipv6.enabled` parameter to `false` to disable an interface.

I changed "or" to "and" for the ipv4 and ipv6 params because in this case you want them both disabled, and clarified that this is used to disable an interface instead of using state: down.

Reply:
Thanks, updated!